app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/OverviewFragment.kt

```diff
@@ -31,9 +30,8 @@ class OverviewFragment : Fragment() {
         return inflater.inflate(R.layout.overview_fragment, container, false)
     }
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
```

Comment:
removing the instantiation of viewmodel in subFragment now that it is initialized in the super class and can be used within subFragments without creating another variable